products/views.py

```diff
@@ -1 +1,23 @@
-from django.shortcuts import render
\ No newline at end of file
+import json
+
+from django.http      import JsonResponse
+from django.views     import View
+
+from products.models import Product, Image
+from biddings.models import Bidding, Order
+
+class ProductDetailGraphView(View):
+    def get(self, request, product_id):
+        try:
+            order = Order.objects.get(id=product_id)
+
+            result = [{
+                'id'         : order.id,
+                'size'       : order.products_sizes.size.name,
+                'price'      : int(order.price),
+                'created_at' : order.created_at.strftime('%Y/%m/%d')
+            } for order in Order.objects.filter(products_sizes__product__id=product_id).order_by('-created_at')]
```

Comment:
```suggestion
            } for order in Order.objects.filter(products_sizes__product_id=product_id).order_by('-created_at')]
```
product__id -> product_id 
_를 하나 줄였습니다. 당연히 _를 두개 사용해서 product의 id를 찾을 수 있습니다. 그런데, products_sizesd에 product_id가 있기 때문에 다음과 같이 수정해도 좋을 것 같습니다.